### DISCUSSION TITLE
Ublock reddit moderators are abusive and remove post for no reason

### DISCUSSION THREAD

**Discussion by RacistandProudofit** [uBlockOrigin/uAssets#23305] [General] [answered]
If this not moderator abuse I don't know what is.  Anyone asking the question "how to block this element in Gmail" gets instantly accused of ban evasion and banned just like this.  Is this an official group because I didn't expect official members of the Ublock team top behave like power trippers like this.

![image](https://github.com/uBlockOrigin/uAssets/assets/167045980/9066e655-1e55-4343-828f-3496f7a488b2)
![image](https://github.com/uBlockOrigin/uAssets/assets/167045980/a5d3dd79-655f-4d11-b479-e6f46386d056)

![image](https://github.com/uBlockOrigin/uAssets/assets/167045980/6c0fb131-2be2-4c5c-ad67-878e6ba3231e)

**Comment by ghost** [comment]
And this one I gave you does not work?

    google.com##body:watch-attr(style)>div[style*="visibility"]:has-text(/google.*?@gmail.com/i)

**Reply by RacistandProudofit** [reply]
No and why do you keep insisting that everyone who complains about this issue as being the same person?  That is ignorant and I would expect more from the uBlock team than these types of accusations.

**Reply by Procyon-b** [reply]
Because it's the same screenshot...
And you wouldn't know that "people" are moderated and banned because of this, unless it's your other accounts. Since this is invisible to anyone outside the mods and the author.

**Reply by RacistandProudofit** [reply, marked as answer]
This screenshot has my account information redacted.  So there is no identifiable information linking me to any of these other posts.  You can't claim that two people complaining about the same problem is sufficient evidence to link the two identities together.  By searching the post history I was able to find posts made by FatNinger69 where they complained about the same thing, and that snob MrDice (whoever that is isn't worthy of the professional Dr title) accused the FatNinger69 guy of being someone else too.  BTW, deleting/banning a user only hides the post from the search feed.  It still shows in third party reddit search tools, google, etc.  So you still don't have any irrefutable evidence to support your claim that I am committing ban evasion.  Furthermore, "ban evasion" isn't even a rule on your stupid subreddit in the first place, as it is only supposed to be enforced by Reddit.  So this notion that I am Chic_Samara98 or FatNinger69 and therefor you have justifiable reasons to ban me is BS because both of those users were not banned by Ublock.  Only the posts made after those accounts were.  So cut the crap.
